GitVersionConfig.yaml

```diff
@@ -1,3 +0,0 @@
-mode: ContinuousDeployment
```

Comment:
Don't remove this file, it's used for GitVersion and we don't want to switch to other tools in the near future.